code/modules/mob/living/simple_animal/revenant/revenant_spawn_event.dm

```diff
@@ -1,3 +1,5 @@
+#define REVENANT_SPAWN_THRESHOLD 20
```

Comment:
20 is a good chunk of most rounds population and is actually getting to the point where you wouldn't want to spawn one. Maybe 10?